src/editorial/web/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Pillars are used for styling
+// RealPillars have Pillar palette colours
+// FakePillars allow us to make modifications to style based on rules outside of the pillar of an article
+type RealPillars = "news" | "opinion" | "sport" | "culture" | "lifestyle"
+type FakePillars = "labs"
+type Pillar = RealPillars | FakePillars
```

Comment:
I'm going to make an argument for the existence of [`ColourScheme`](https://github.com/guardian/types/blob/master/ColourScheme.ts) again here 🙂

Reply:
I'm going to happily agree with you 🙂